model/pdata/common.go

```diff
@@ -193,6 +206,13 @@ func (a AttributeValue) ArrayVal() AnyValueArray {
 	return newAnyValueArray(&arr.Values)
 }
 
+// BytesVal returns the []byte value associated with this AttributeValue.
+// If the Type() is not AttributeValueTypeBytes then returns false.
+// Calling this function on zero-initialized AttributeValue will cause a panic.
+func (a AttributeValue) BytesVal() []byte {
```

Comment:
Comment that it is discouraged to modify the array in-place.

Reply:
Added.